lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -125,6 +130,16 @@ export class LogCreationModel extends Observable {
             options.headers['Content-Type'] = 'application/json';
         }
 
+        // The field detectorOrSubsystem and issueDescription should be allowed only if the log templateKey is On-call
+        if (templateKey) {
```

Comment:
Are you sure this is the right place to put this?

Reply:
done